src/sentry/static/sentry/app/views/releases/list/index.tsx

```diff
@@ -374,11 +374,14 @@ class ReleasesList extends AsyncView<Props, State> {
 }
 
 const SortAndFilterWrapper = styled('div')`
-  display: grid;
-  grid-gap: ${space(2)};
+  display: inline-flex;
+  flex-wrap: wrap;
+  gap: ${space(2)};
```

Comment:
This is not yet working on some major browsers, for example mobile safari:

![IMG_7924](https://user-images.githubusercontent.com/9060071/106105675-a3e07a00-6144-11eb-94b2-d2ba7feea6fd.PNG)

Reply:
yes, I also noticed that